### PR TITLE
Feat/question service

### PR DESCRIPTION
**Pull request by lsyurea**
Create scaffold for backend services for question routes.

Routes that are created from this service include:
- GET /questions/ : Get all questions
- GET /userQuestions/ : Get all user questions

More information on local set up with docker can be found in teh readme.

### CODE REVIEW

File: question-service/.husky/pre-commit
Comment (Kurtyjlee) [inline question-service/.husky/pre-commit:1]: Eh nice one! If you want, this file can be extended to use prettier as well

File: question-service/src/db/mongoClient.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { MongoClient, ServerApiVersion, Db } from 'mongodb';
+import * as dotenv from 'dotenv';
+// configure env variables
+dotenv.config({ path: '.env.dev' });
+
+const MONGODB_URI = process.env.MONGODB_CONNECTION_STRING || '';
+// console.log(MONGODB_URI);
```
Comment (Kurtyjlee) [inline question-service/src/db/mongoClient.ts:7]: I think this comment should be removed

**Review by Kurtyjlee — APPROVED** [review]
Some minor nits, else LGTM

File: question-service/src/db/mongoClient.ts (hunk 1 above)
Comment (lsyurea) [reply question-service/src/db/mongoClient.ts:7]: Yups I agree

File: question-service/.husky/pre-commit
Comment (lsyurea) [reply question-service/.husky/pre-commit:1]: Yes, it would be better

Comment (Kurtyjlee) [reply question-service/.husky/pre-commit:1]: We can do it in the next iterations, no rush

**Review by Kurtyjlee — APPROVED** [review]
LGTM

**Pull request merged**